libs/design/src/molecules/list/list/list.component.ts

```diff
@@ -32,4 +51,26 @@ export class DaffListComponent {
   @HostBinding('class.daff-list--navigation') get navigation() {
     return this.mode === DaffListModeEnum.Navigation;
   }
+  
+  get listType(): DaffListType {
+    return this._getHostElement().localName;
+   }
+
+  constructor(private elementRef: ElementRef) {}
+
+
+  @HostBinding('class.daff-nav-list') get nav() {
```

Comment:
Needs a test covering the class.

Reply:
The test is already written, but I'll move it up to be directly under the daff-list class test.